man/systemd.exec.xml

```diff
@@ -1156,9 +1156,9 @@ BindReadOnlyPaths=/var/lib/systemd</programlisting>
         <citerefentry><refentrytitle>unshare</refentrytitle><manvolnum>2</manvolnum></citerefentry>,
         <citerefentry><refentrytitle>clone</refentrytitle><manvolnum>2</manvolnum></citerefentry> and
         <citerefentry><refentrytitle>setns</refentrytitle><manvolnum>2</manvolnum></citerefentry> system calls, taking
-        the specified flags parameters into account. Note that — if this option is used — in addition to restricting
-        creation and switching of the specified types of namespaces (or all of them, if true) access to the
-        <function>setns()</function> system call with a zero flags parameter is prohibited.  This setting is only
+        the specified flags or nstype parameters into account. Note that — if this option is used — in addition to
```

Comment:
well, the "flags" refers not to the name of the parameter, but to the concept of it... i.e. regardless whether it's named "flags" or not, it's a flags parameter, as it generally is the OR combination of a number of flag values... That's also the reason why the flags is not contained in `<varname>`, as we'd do it if this was the parameter name...

Reply:
true, setns() inherits the parameter from clone() i guess, but in contrast to clone only accepts a subset of the possible flags and only one set at a time... So yeah, this is indeed a bit confusing...

Maybe the first time we could refer to `the flags parameter <varname>nsflags</varname>`? And the second time just `<varname>nsflags</varname>` should be fine. That way, we are verbose enough for everybody to grok what's going on…

Reply:
Hm, I don’t know, won’t that first time be confusing for `clone` and `unshare`, where the parameter has a different name?

To be honest, right now I kinda feel like just closing the pull request. The documentation is okay as it is, and I’m not sure how to improve it. (When I started with this pull request, I’d temporarily forgotten about `clone` and `unshare`, so the `nstype` name in `setns` was more important to me… with `clone`/`unshare`’s `flags` in the mix as well, the way forward is less clear to me.)